<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IN-PG02P63S" failures="1" tests="1" name="com.AssignmentSelenium.tests.NewReleasesTest" time="7.925" errors="0" timestamp="2022-03-07T20:25:08 IST" skipped="0">
  <testcase classname="com.AssignmentSelenium.tests.NewReleasesTest" name="NewReleases" time="7.925">
    <failure message="expected [Buy Safari Ray Voyage Trolley Bag Small and Medium Size, 53 cms and 67 Blue/Cyan Printed Hard Side Travel Bag for Men and Women, 4 Wheel Luggage Suitcase for Travelling, (RAYVOYAGESM4WPRN) at Amazon.in] but found [Buy WILDHORN Top Grain Leather Wallet for Men | Ultra Strong Stitching | Handcrafted | RFID Blocking | Zip Wallet with 9 Card Slots | 2 ID Slots (Black) at Amazon.in]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Buy Safari Ray Voyage Trolley Bag Small and Medium Size, 53 cms and 67 Blue/Cyan Printed Hard Side Travel Bag for Men and Women, 4 Wheel Luggage Suitcase for Travelling, (RAYVOYAGESM4WPRN) at Amazon.in] but found [Buy WILDHORN Top Grain Leather Wallet for Men | Ultra Strong Stitching | Handcrafted | RFID Blocking | Zip Wallet with 9 Card Slots | 2 ID Slots (Black) at Amazon.in]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.AssignmentSelenium.tests.NewReleasesTest.NewReleases(NewReleasesTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- NewReleases -->
  <system-out/>
</testsuite> <!-- com.AssignmentSelenium.tests.NewReleasesTest -->
